---
namespace: Compile
expectation: Fail
outputs:
  - "Error [EASG0373015]: a function named \"main\" already exists in this scope\n    --> compiler-test:9:1\n     |\n   9 | function main(y: bool) -> bool {\n  10 |      ...\n  11 |      ...\n  12 | }\n     | ^"
